### PR TITLE
Configurable gas costs per individual WASM opcodes

### PR DESCRIPTION
**Pull request by camilbancioiu**
* Defines struct `WASMOpcodeCost`, which stores gas cost for each of the 410 WASM opcodes
* Handles loading the `WASMOpcodeCost` struct from a `gasMap`
* Handles converting the `WASMOpcodeCost` to an array of 410 `uint32` elements, to pass to `go-ext-wasm` for creating a metered Wasmer instance

### CODE REVIEW

File: arwen/context/arwen.go [hunk 1]
```diff
@@ -226,6 +227,14 @@ func (host *vmContext) RunSmartContractCall(input *vmcommon.ContractCallInput) (
 	return vmOutput, nil
 }
 
+func opcode_costs_uniform_value(value uint32) *[wasmer.OPCODE_COUNT]uint32 {
```
Comment (sasurobert) [inline arwen/context/arwen.go]: function not used

File: go.mod [hunk 2]
```diff
@@ -5,7 +5,8 @@ go 1.13
 require (
 	github.com/ElrondNetwork/elrond-vm-common v0.0.9
 	github.com/ElrondNetwork/elrond-vm-util v0.0.5
-	github.com/ElrondNetwork/go-ext-wasm v0.0.6
+	github.com/ElrondNetwork/go-ext-wasm v0.0.7
+	github.com/ElrondNetwork/managed-big-int v0.0.3
```
Comment (sasurobert) [inline go.mod]: why managed-big-int ? it is not used.

File: arwen/context/arwen.go (hunk 1 above)
Comment (camilbancioiu) [reply arwen/context/arwen.go]: Removed.

File: go.mod (hunk 2 above)
Comment (camilbancioiu) [reply go.mod]: Vestigial. Removed.

**Pull request merged**